pulp_ansible/app/models.py

```diff
@@ -59,6 +67,9 @@ def __str__(self):
 
     class Meta:
         unique_together = ("namespace", "name")
+        permissions = [
+            ("manage_roles_collection", "Can manage roles on Collections"),
```

Comment:
I don't know that we need to manage roles on a collection level. This will typically be handled at the namespace or repository level.

@mdellweg thoughts?

Reply:
@mdellweg  answered this above, I missed removing this instance.